airflow-client/airflow-client.go

```diff
@@ -213,7 +213,7 @@ func (c *HTTPClient) DoAirflowClient(doOpts *httputil.DoOptions) (*Response, err
 		}
 	}
 
-	response, err := c.Do(doOpts)
+	response, err := c.Do(doOpts) //nolint:bodyclose
```

Comment:
@neel-astro @jeremybeard @pritt20 I ended up with a lot of these `//nolint:bodyclose` exceptions as I was making the updated linter setup happy. I really don't understand why its complaining though. In every single case we have a `defer response.Body.Close()` type statement right after the error check. I don't know how to make it happy haha. If anyone has an idea here, please let me know. Otherwise, I think we just ignore these. We could keep the in-line versions, or we could just disable `bodyclose` in the linter completely. Curious what you think.

Reply:
Yeah seems to be misfiring. I would remove the linter entirely.

Reply:
Cool, dropped all the in-line ignore statements and just disabled the linter.